src/main/java/seedu/duke/option/OptionList.java

```diff
@@ -0,0 +1,37 @@
+package seedu.duke.option;
+
+import java.util.ArrayList;
+
+public class OptionList implements OptionListInterface {
+    private final ArrayList<OptionInterface> optionList;
+    private int optionCount;
```

Comment:
Good idea but may be redundant as we can call "length()" on the ArrayList

Reply:
Ah yes, this makes sense. We can cut down on unnecessary variables this way!

Reply:
Ahh, slipped my mind! Thank you for pointing it out!